data/PostData.json

```diff
@@ -0,0 +1,56 @@
+
+{
+  "post": {
+    "id": 1,
+    "user": {
+      "id": 1,
+      "name": "John Doe",
+      "username": "johndoe",
+      "profilePicture": "path/to/profile_picture.jpg"
+    },
+    "content": [
```

Comment:
the content will be an HTML string

Reply:
What does that mean can you explain it more, please?